charts/ks-devops/templates/_helpers.tpl

```diff
@@ -11,16 +11,15 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 If release name contains chart name it will be used as a full name.
 */}}
 {{- define "ks-devops.fullname" -}}
-{{- if .Values.fullnameOverride }}
-{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" }}
-{{- else }}
-{{- $name := default .Chart.Name .Values.nameOverride }}
-{{- if contains $name .Release.Name }}
-{{- .Release.Name | trunc 63 | trimSuffix "-" }}
-{{- else }}
-{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
+{{ .Release.Name | trunc 63 | trimSuffix "-" }}
 {{- end }}
+
+{{- define "ks-devops-apiserver.fullname" -}}
```

Comment:
I think `ks-devops.apiserver-fullname` is more appropriate than `ks-devops-apiserver.fullname`.